docs/protocol.md

```diff
@@ -590,6 +591,10 @@ Value is string "host:port".
 
 Send claim message to `host:port` when response is being sent.
 
+### Transport Header `cn` -- Caller Name
+
+Value is name of the service making the call.
```

Comment:
Yeah, there may be multiple services registering or calling on a single connection.

Certainly this will be the case for internal hyperbahn hopes (Entry -> Exit, Exit -> CalledService).